beetsplug/lyrics.py

```diff
@@ -77,6 +78,11 @@ class HTMLParseError(Exception):
 
 # Utilities.
 
+def unichar(i):
+    try:
+        return six.unichr(i)
+    except ValueError:
+        return struct.pack('i', i).decode('utf-32')
```

Comment:
Is there a particular reason why this is necessary? Are some files encoded with UTF-32 (which is a fairly rare format)?